eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala

```diff
@@ -351,6 +359,7 @@ object Setup {
 }
 
 case class Kit(nodeParams: NodeParams,
+               bitcoin: BitcoinJsonRPCClient,
```

Comment:
I'm not a big fan of exposing that to all plugins.

The only reason it's necessary is because you put some of the logic inside `Eclair.scala` (in the `globalBalance()` function). I think it wasn't a good choice, IMO the logic should be inside the balance actors, and `globalBalance()` should simply send a request to the actor and return its response (this way it doesn't make direct calls to the bitcoin client).

Reply:
That's fair, I'll rework that.

Reply:
Done with 3606f4f.

Reply:
Done in 3606f4f.